pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -93,6 +93,8 @@ type PostgresSpec struct {
 	// deprecated json tags
 	InitContainersOld       []v1.Container `json:"init_containers,omitempty"`
 	PodPriorityClassNameOld string         `json:"pod_priority_class_name,omitempty"`
+
+	TopologySpreadConstraints []v1.TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty"`
```

Comment:
when introducing new fields to the api you have to update the CRDs, too. Unfortunately, we don't use a modern framework where this can be generated and you have to the [manifests](https://github.com/zalando/postgres-operator/blob/master/manifests/postgresql.crd.yaml) and in [go code](https://github.com/zalando/postgres-operator/blob/master/pkg/apis/acid.zalan.do/v1/crds.go#L118).

Plus, you need to run code generation -> ./hack/update-codegen.sh

Reply:
I updated.